pxb-tag/PUBLISHING.md

```diff
@@ -0,0 +1,11 @@
+# Publishing Instructions
+
+To update the version number, edit the version in `package.json`.
+
+To publish a new package through NPM, run the following commands from the root folder:
+
+```
+npm publish --tag <alpha | beta>
```

Comment:
weren't we going to update this to be a publish:package command?

Reply:
You're right, I've added the tag script but forgot to include the updated PUBLISHING instructions.   Small PR incoming...